src/test/java/persistence/sql/dml/InsertQueryBuilderTest.java

```diff
@@ -0,0 +1,23 @@
+package persistence.sql.dml;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import persistence.entity.notcolumn.Person;
+import persistence.sql.dml.InsertQueryBuilder;
+
+import static persistence.sql.dml.TestFixture.person_철수;
+
+class InsertQueryBuilderTest {
+    @Test
+    @DisplayName("[요구사항 1] 컬럼 전체에 대해 insert 쿼리를 구현하라")
+    void 요구사항1_1_test() {
+        //given
+        String expectedQuery = "INSERT INTO users (nick_name,old,email) VALUES ('김철수',21,'chulsoo.kim@gmail.com')";
+        // when
+        String actualQuery = new InsertQueryBuilder(Person.class).getInsertQuery(person_철수);
```

Comment:
```suggestion
        String actualQuery = new InsertQueryBuilder<Person>(Person.class).getInsertQuery(person_철수);
```
제네릭을 사용해주셨지만 생성시에는 빠져있네요.

추가적으로
어떠한 부분때문에 제네릭을 넣으신건지 궁금해요!

Reply:
getInsertQuery의 파라메터 타입이 유연했으면 좋겠다는 생각 때문에 넣었습니다-! 지금 보니 Object만 넣어도 충붕할것 같네요!